src/ansys/mapdl/core/mapdl.py

```diff
@@ -491,13 +491,28 @@ def queries(self):
     def non_interactive(self):
         """Non-interactive context manager.
 
-        Use this when using commands that require user
-        interaction within MAPDL (e.g. :func:`Mapdl.vwrite`).
+        Allows to execute code without user interaction or waiting
+        between PyMAPDL responses.
+        It can be also used to execute some commands which are not
```

Comment:
```suggestion
        It can also be used to execute some commands which are not
```
```suggestion
        It can be also used to execute some commands which are not
```